### DISCUSSION TITLE
Welcome to multicorn2 Discussions!

### DISCUSSION THREAD

**Discussion by luss** [pgsql-io/multicorn2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ShaheedHaque** [comment]
Well done on getting multicorn2 going (I was one of those trying to help get support for newer PG releases going, so I wish you lots of luck). BTW, assuming you plan to do releases via pypi.org, is there any chance of keep the package name intact?

**Comment by court-jus** [comment]
Hi, I'm Ghislain, from south France, and work using both PostgreSQL and ES.
